Cargo.toml

```diff
@@ -59,12 +62,14 @@ env_logger = "0.8"
 
 [features]
 # actix-web support
-actix = ["actix3"]
-actix-nightly = ["actix3-nightly"]
+actix = ["actix4"]
```

Comment:
I guess this means actix4 is a breaking change?

Reply:
Yeah, I think the same breaking change was introduced when changing from 2 to 3.

Reply:
I think we should change this going forward, otherwise we'll keep having breaking changes.

Also, we have a problem for testing, example one of the tests requires actix:
```
required-features = ["cli", "actix", "uuid", "chrono", "swagger-ui"]
```
This means we don't run the tests for previous versions, which seems wrong, and explains why we had a few actix2 bugs that should have been caught.

I propose that the actix version should be explicitly specified in the feature: "actix3" or "actix4".
Then we can have an "actix" feature to allow tests to run for all actix versions - which could just be the "actix-base" we already have:
```
required-features = ["cli", "actix-base", "uuid", "chrono", "swagger-ui"]
```

Reply:
We do test previous versions now. I've added it in this PR.
Of course I can remove bare 'actix' feature, but this will be breaking change too. The last one though :-) 
PS. Changing 'actix' feature to take a role of 'actix-base' will be a "silent braking change" so I'm against it.
What do you think?

Reply:
We do test previous but not for test_app, because "actix" is a required feature for this test to run, which means test_app tests don't run for previous versions?

Reply:
Ok, I didnt understand you in the first place. So, things to be done:

1. Remove 'actix' alias-feature
2. Do tests for actix-base

Confirm and I'll amend.

Reply:
Sorry about that, eloquence was never my strong suite :D 
Yep I'm happy with that!